eng/scripts/Create-APIReview.ps1

```diff
@@ -0,0 +1,40 @@
+Param(
+    [Parameter(Mandatory=$True)]
+    [string] $ArtifactName,
+    [Parameter(Mandatory=$True)]
+    [string] $OutPath,
+    [Parameter(Mandatory=$True)]
+    [string] $ApiviewUri,
+    [Parameter(Mandatory=$True)]
+    [string] $ApiKey,
+    [Parameter(Mandatory=$True)]
+    [string] $ApiLabel,
+    [Parameter(Mandatory=$True)]
+    [string] $SourceBranch,
+    [Parameter(Mandatory=$True)]
+    [string] $DefaultBranch,
+    [Parameter(Mandatory=$True)]
+    [string] $ConfigFileDir
+)
+
+Write-Host "$PSScriptRoot"
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+$createReviewScript = (Join-Path $PSScriptRoot .. common scripts Create-APIReview.ps1)
+Set-Location $PSScriptRoot
+
+Write-Host "Creating API review artifact for $ArtifactName"
+New-Item -ItemType Directory -Path $OutPath/$ArtifactName -force
+
+$gitroot = Join-Path $PSScriptRoot .. ..
+Write-Host "Get-ApiViewCommandLine.ps1 $gitroot $ArtifactName"
+$cmdLine = & $PSScriptRoot/Get-ApiViewCommandLine.ps1 $gitroot $ArtifactName
+Write-Host "Executing clang++ command:"
+Write-Host $cmdLine
+$cmd, $cmdArgs = $cmdLine -split ' '
```

Comment:
Does this correctly put all the rest of the args in the second variable or only the second one?

Reply:
Yes. See the second example [here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_assignment_operators?view=powershell-7.2#assigning-multiple-variables).